content/sensu-go/5.0/installation/upgrade.md

```diff
@@ -0,0 +1,109 @@
+
+Sensu Go includes important changes to all parts of Sensu: architecture, installation, resource definitions, event data model, check dependencies, filter evaluation, and more.
+Sensu Go also includes a lot of powerful features to make monitoring easier to build, scale, and offer as a self-service tool to your internal customers.
+
+- [Packaging](#packaging)
+- [Architecture](#architecture)
+- [Entities](#entities)
+- [Checks](#checks)
+- [Events](#events)
+- [Handlers](#handlers)
+- [Filters](#filters)
+- [Assets](#assets)
+- [Role-based access control](#role-based-access-control)
+- [Silencing](#silencing)
+- [Token substitution](#token-substitution)
+- [Aggregates](#aggregates)
+- [API](#api)
+
+### Packaging
+Sensu is now provided as three packages: sensu-backend, sensu-agent, and sensu-cli (sensuctl).
+This results in a fundamental change in Sensu terminology from Sensu Core: the server is now the backend; the client is now the agent.
+To learn more about new terminology in Sensu Go, see the [glossary][1].
+
+### Architecture
+The external RabbitMQ transport and Redis datastore in Sensu Core have been replaced with an embedded transport and [etcd datastore][2] in Sensu Go.
+The Sensu backend and agent are configured using YAML files or using the `sensu-backend` or `sensu-agent` command-line tools, instead of using JSON files.
+Sensu checks and pipeline elements are now configured via the API or sensuctl tool instead of JSON files.
+See the [backend][3], [agent][4], and [sensuctl][5] reference docs for more information. 
+
+### Entities
+“Clients” are now represented within Sensu Go as abstract “entities” that can describe a wider range of system components (network gear, web server, cloud resource, etc.)
+Entities include “agent entities” (entities running a Sensu agent) and familiar “proxy entities”.
+See the [entity reference][6] and the guide to [monitoring external resources][7] for more information.
+
+### Checks
+Standalone checks are no longer supported in Sensu Go, although similar functionality can be achieved using role-based access control, assets, and entity subscriptions.
+There are also a few changes to check definitions to be aware of. The `stdin` check attribute is no longer supported in Sensu Go, and Sensu Go no longer tries to run a "default" handler when executing a check without a specified handler. Additionally, round-robin subscriptions are not yet available in Sensu Go 5.0
+
+[Check hooks][8] are now a resource type in Sensu Go, meaning that hooks can be created, managed, and reused independently of check definitions.
+You can also execute multiple hooks for any given response code.
+
+### Events
+All check results are now considered events and are processed by event handlers.
+You can use the built-in [incidents filter][9] to recreate the Sensu Core behavior in which only non-zero check results are considered events.
```

Comment:
how about rewording this sentence like so:

"You can use the built-in incidents filter to recreate the Sensu Core behavior in which only check results with a non-zero status are considered events."